firebase-config/src/test/java/com/google/firebase/remoteconfig/FirebaseRemoteConfigTest.java

```diff
@@ -196,6 +205,7 @@ public void setUp() throws Exception {
   public void ensureInitialized_notInitialized_isNotComplete() {
     loadCacheWithConfig(mockFetchedCache, /*container=*/ null);
     loadCacheWithConfig(mockDefaultsCache, /*container=*/ null);
+    when(mockFirebaseInstanceId.getInstanceId()).thenReturn(Tasks.forResult(INSTANCE_ID_RESULT));
```

Comment:
in previous android changes tests, we have used methods like `loadInstanceIdAndToken()` (which you changed below). WDYT about doing something similar in this file? It does make it slightly more readable, but at this point, i'm just infavor of consistency :)  https://osscs.corp.google.com/firebase/firebase-android-sdk/+/master:firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java;l=868-871;drc=5301121df4727b357293b3b7de00555362f90170

Reply:
Sure, I like the consistency. I do think just having the mock in the method is easier to read, though.